src/api/app/controllers/webui/project_controller.rb

```diff
@@ -239,8 +239,39 @@ def add_group
 
```

Comment:
What typo? In what commit? You mean in this one? "[webui] Refactor ProjecController#buildresult"